common/src/View/EntityAttributeGrid.cpp

```diff
@@ -227,18 +231,40 @@ namespace TrenchBroom {
         class EntityAttributeCellEditor : public wxGridCellTextEditor
         {
         private:
+            EntityAttributeGrid* m_grid;
             EntityAttributeGridTable* m_table;
+            int m_row, m_col;
             
         public:
-            EntityAttributeCellEditor(EntityAttributeGridTable* table)
-            : m_table(table) {}
+            EntityAttributeCellEditor(EntityAttributeGrid* grid, EntityAttributeGridTable* table)
+            : m_grid(grid),
+            m_table(table),
+            m_row(-1),
+            m_col(-1) {}
 
+        private:
+            void OnCharHook(wxKeyEvent& event) {
+                if (event.GetKeyCode() == WXK_TAB) {
+                    // HACK: Consume tab key and use it for cell navigation.
+                    // Otherwise, wxTextCtrl::AutoComplete uses it for cycling between completions (on Windows)
+                    m_grid->tabNavigate(m_row, m_col, !event.ShiftDown());
+                    return;
+                }
```

Comment:
Please put this in an else block and remove the trailing return in the if block.